api_app/models/domain/restricted_resource.py

```diff
@@ -9,7 +9,7 @@ class RestrictedProperties(AzureTREModel):
     description: str = ""
     overview: str = ""
     connection_uri: str = ""
-    internal_connection_uri: str = ""
+    is_exposed_externally: bool = True
```

Comment:
This is the default when creating the object, so if its not defined if it is exposed or not the UI will presume it is and provide a connect button if there is also a `connection_uri` specified, rather than the UI dialog to copy the URI. 

Not sure it really matters to be honest. Ideally bundles should define a `connection_uri` and `is_exposed_externally` and not use the default, but the default is more for backwards compatibility where `is_exposed_externally` has not been defined.